src/dfx/src/lib/operations/cmc.rs

```diff
@@ -12,6 +12,8 @@ use ic_agent::Agent;
 const NOTIFY_CREATE_CANISTER_METHOD: &str = "notify_create_canister";
 const NOTIFY_TOP_UP_METHOD: &str = "notify_top_up";
 
+pub const MEMO_CREATE_CANISTER: u64 = 1095062083_u64;
```

Comment:
duplicate of https://github.com/dfinity/sdk/blob/master/src/dfx/src/commands/ledger/create_canister.rs#L16

Reply:
Thanks, though please see https://github.com/dfinity/sdk/pull/3259/files#diff-f0ee04621aed1a936f3a09e642c3cc08b34d182d9314ed7ee62f3e69a98c39fbL16 , which deletes that